Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/local/source/LocalStatisticsProvider.kt

```diff
@@ -41,23 +43,35 @@ class LocalStatisticsProvider @Inject constructor(
         }
     }
 
-    val current: Flow<List<StatisticsData>> = localStatisticsData.data
+    val current: Flow<LocalStatisticsData> = localStatisticsData.data.map {
```

Comment:
When I select  "Nordrhein-Westfallen>Aachen (Stadtregion)", nothing happens. :thinking:

Reply:
Düssesldorf also  doesn't work, all items from NRW?

Reply:
maybe some id mismatch in the test data, will investigate

Reply:
the mapping for nrw to package id was indeed wrong, fixed it :)